python/paddle/v2/framework/tests/test_reshape_op.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import numpy as np
+from gradient_checker import GradientChecker, Operator
+from op_test_util import OpTestMeta
+
+
+class TestReshapeOp(unittest.TestCase):
+    __metaclass__ = OpTestMeta
+
+    def setUp(self):
+        self.type = "reshape"
+        self.inputs = {'X': np.random.random((37, 51)).astype("float32"), }
+        self.attrs = {'shape': [51, 37]}
+        self.outputs = {'Out': self.inputs['X'].reshape(self.attrs['shape'])}
+
+
+class ReshapeGradOpTest(GradientChecker):
+    def test_normal(self):
```

Comment:
now can use the function `setUp` to generate forward operator.
please take a look at [this case](https://github.com/PaddlePaddle/Paddle/pull/3888/files#diff-c2dd71df56cae469f7474f18bea1d8a0R70)

Reply:
Done